webpack/assets/javascripts/react_app/components/notifications/NotificationDrawerToggle.js

```diff
@@ -21,7 +21,7 @@ class NotificationDrawerToggle extends Component {
     switch (actionType) {
       case ACTIONS.RECEIVED_NOTIFICATIONS: {
         this.setState({
-          count: NotificationsStore.getNotifications().length,
+          unreadNotificationsCount: NotificationsStore.hasUnreadNotifications(),
```

Comment:
@lizagilman great. Have a look at NotificationPanel.js Consider extracting the calculation of the unread notifications to a helper or services file and reuse for your purposes.

```js
    let unread;

    if (notifications) {
      unread = notifications.reduce((total, curr) => {
        if (!curr.seen) {
          total = total + 1;
        }
        return total;
      }, 0);
    } else {
      unread = 0;
    }
```